### DISCUSSION TITLE
Cscope support ?

### DISCUSSION THREAD

**Discussion by cnagesh-arrcus** [helix-editor/helix#4296] [Ideas] [answered]
Any plans to implement cscope support ? I don't see any mention of this in docs.

**Comment by the-mikedavis** [comment, marked as answer]
This is similar to ctags https://github.com/helix-editor/helix/discussions/3518

There isn't any support currently and I wouldn't expect any except maybe as a plugin - LSP covers the same features and is language agnostic.

**Reply by cnagesh-arrcus** [reply]
Thanks
